encrpt/aes_ofb_test.go

```diff
@@ -0,0 +1,191 @@
+package encrpt
+
+import (
+	"testing"
```

Comment:
Check all the test file.
We should init config module first,cause the encrypt module depends on the log module,which depends on the config module.